config/cars/kunos/ks_toyota_ae86_drift.ini

```diff
@@ -1,2 +1,8 @@
 [INCLUDE]
-INCLUDE=ks_toyota_ae86.ini
\ No newline at end of file
+INCLUDE=ks_toyota_ae86.ini
+
+[CustomEmissive]
+Meshes= GEO_fbumper_glass_drift_SUB0
```

Comment:
i hope this wont cause any issues, the drift front bumper uses seperate materials to the normal AE86, i dont think duplcating the configs would be necesarry here.